tests/operations.go

```diff
@@ -364,6 +364,92 @@ func (ops *Operations) IsPodRunningEventually(namespace, podName string) bool {
 		Should(BeTrue())
 }
 
+// ExecuteZpoolCMDEventually executes the zpool command in cstor pool container
+// and returns stdout
+func (ops *Operations) ExecuteZpoolCMDEventually(podObj *corev1.Pod, cmd string) string {
+	var err error
+	output := &pod.ExecOutput{}
+	podName := podObj.Name
+	namespace := podObj.Namespace
+	status := ops.IsPodRunningEventually(namespace, podName)
+	Expect(status).To(Equal(true),
+		"while checking the status of pod {%s} in namespace {%s}",
+		podName,
+		namespace,
+	)
+	for i := 0; i < maxRetry; i++ {
+		output, err = ops.PodClient.WithNamespace(namespace).
+			Exec(
+				podName,
+				&corev1.PodExecOptions{
+					Command: []string{
+						"/bin/bash",
+						"-c",
+						cmd,
+					},
+					Container: podObj.Spec.Containers[0].Name,
+					Stdin:     false,
+					Stdout:    true,
+					Stderr:    true,
+				},
+			)
+		Expect(err).ShouldNot(
+			HaveOccurred(),
+			"while geting the zpool pool guid of pod {%s}",
```

Comment:
correct the error message.. instead of 'zpool pool guid', you can print the command that is given as argument

Reply:
Oops I will update it